catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/FilterNodeMapImpl.java

```diff
@@ -69,7 +70,13 @@ public String getOperator() {
         .map(List.class::cast)
         .flatMap(List::stream)
         .map(Map.class::cast)
-        .map(FilterNodeMapImpl::new)
+        .map(
+            new Function<Map, FilterNodeMapImpl>() {
+              @Override
+              public FilterNodeMapImpl apply(Map json1) {
+                return new FilterNodeMapImpl(json1);
+              }
+            })
```

Comment:
❓ Can this be collapsed down? 
```
.map(json -> new FilterNodeMapImpl(json))
```

Actually, shouldn't the old syntax still work? 
```
.map(FilterNodeMapImpl::new)
```

Reply:
done